dissect/target/loaders/asdf.py

```diff
@@ -23,11 +23,4 @@ def map(self, target):
         for disk in self.asdf.disks():
             target.disks.add(AsdfContainer(disk))
 
-        overlay = target.fs.add_layer()
-        for member in self.asdf.metadata.members():
-            if member.isdir():
-                continue
-
-            path = fsutil.join(self.METADATA_PREFIX, member.name)
-            entry = TarFile(overlay, path, member.name, self.asdf.metadata.tar)
-            overlay.map_file_entry(entry.path, entry)
+        target.fs.mount(self.METADATA_PREFIX, TarFilesystem(self.asdf.open(IDX_METADATA)))
```

Comment:
I cannot comment on lines that have not changed, but since we are updating the ADSF-loader maybe it is a good idea to also add the types?